src/config.ts

```diff
@@ -30,9 +30,9 @@ export const UNIT_BASE_HEALTH = 40;
 export const UNIT_BASE_MANA = 60;
 export const UNIT_BASE_STAMINA = 300;
 // For game difficulty, I'm making the attack range less than the unit base stamina
-export const PLAYER_BASE_ATTACK_RANGE = UNIT_BASE_STAMINA * 0.8;
+export const PLAYER_BASE_ATTACK_RANGE = 200; //previously 240 | UNIT_BASE_STAMINA * 0.8
```

Comment:
This will probably make the early game harder, but I think I like that.  It'll also make players much more inclined to choose range upgrades

Reply:
I'll have to play with it a bit before approving, but intuitively, I like this change.

Reply:
To add to my reasoning, I initially made player attack range 80% of unit base stamina so that it would be possible for golems to be out of your range when you end your turn and still hit you on their turn.  So you have to step into danger in order to target them.  This change increases that and I feel like new players find the game to be hard until they get the hang of it - so I'm more interested in making the game harder for established players than new players, but regardless, I will try this out and see what I think.  Just wanted to add some historical context

Reply:
That makes sense! The very first level is definitely made more difficult with this change. Interesting to think about.

I agree the game should be more difficult for experienced players, and also agree that we don't want to scare new players off. I will have to keep that second part in mind when thinking about difficulty.

Reply:
Agreed,  revert the PLAYER_BASE_ATTACK_RANGE change and keep the rest and I'll merge it 👍